bank_transactions.rb

```diff
@@ -0,0 +1,35 @@
+def bank
+  logged_in = true
+  balance = 400
+
+  while logged_in
+    puts "Would you like to make a 'deposit', 'withdraw', or see your 'balance'?"
+    user_input = gets.chomp.downcase
+    method = user_input
+
+    if method == 'balance'
+      puts "Current balance: #{balance}"
+    else
+      puts "How much would you like to " + method + "?"
+      num = gets.chomp.to_i
+
+      if method == 'deposit'
+        balance += num
+        puts "Current balance: #{balance}"
+      elsif method == 'withdraw'
+        balance -= num
+        puts "Current balance: #{balance}"
+      end
+    end
+    puts "Would you like to make another transaction?"
+    answer = gets.chomp
+    if answer == 'yes'
+      logged_in = true
+    else
+      logged_in = false
+    end
+  end
+end
+
+puts "Welcome to the Bank of Katlyn"
+bank
```

Comment:
This looks good Katlyn, but I would suggest rewriting it as a Class. For example:
```
Class Bank
   attr_accessor :balance
  def initialize (balance)
    @balance = balance
  end
# add your methods for deposit, withdraw, and check balance here
end
```